src/tr/tonizutoon/src/eu/kanade/tachiyomi/extension/tr/tonizutoon/TonizuToon.kt

```diff
@@ -17,4 +20,16 @@ class TonizuToon : Madara(
     override val mangaDetailsSelectorAuthor = ".summary-heading:contains(Yazar) ~ .summary-content"
 
     override val mangaDetailsSelectorStatus = ".summary-heading:contains(Durumu) ~ .summary-content"
+
+    override val client = network.cloudflareClient.newBuilder()
+        .addNetworkInterceptor(::loginCheckInterceptor)
+        .build()
+
+    private fun loginCheckInterceptor(chain: Interceptor.Chain): Response {
+        val request = chain.request()
+        if (request.url.encodedPath == "/giris-uyari/") {
```

Comment:
should the response url be checked? in case it redirects. since this url wouldn't be directy called in a request

Reply:
Since it's used as a network interceptor it will catch it when trying to navigate to the redirected URL